cluster/node-pools/master-default/userdata.yaml

```diff
@@ -491,7 +489,7 @@ write_files:
               -> disableAccessLog()
               -> setPath("/metrics")
               -> "http://127.0.0.1:8081";
-            routegroups_admission_webhook: Path("/routegroups-admission-webhook")
+            skipper_admission_webhook: Path("/skipper-admission-webhook")
```

Comment:
Check out where else `/routegroups-admission-webhook` is used, I think you'd need to update there as well.